template/README.t.md

```diff
@@ -1,12 +1,10 @@
 # PLUGIN_TITLE
 
-[PostCSS] plugin PLUGIN_DESC.
-
-[PostCSS]: https://github.com/postcss/postcss
+${description}.
```

Comment:
Why we changed description format and removed link to PostCSS?

Reply:
It's the restriction from `prompts`, I can't figure out a way to preserve the leading words.

https://github.com/terkelg/prompts#options

All others are fixed.

Reply:
Replace with `enquirer` snippet, which is perfect for this: https://github.com/enquirer/enquirer#snippet-prompt